.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: scala
+
+scala:
+  - 2.12.10
+  - 2.13.1
+jdk:
+  - openjdk8
+  - openjdk11
+
+install: 
+  - ./gradlew --version
+  - gradle --version
```

Comment:
I don't think we need this line since I'm guessing it would only print the version of Gradle that's pre-installed on Travis which we don't want to rely on.

Reply:
I will remove.  I added it to show that it is an unreliably old version.  I should have added a comment as such.